libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -229,6 +229,7 @@
 #define HAL_FLOW_PX4_MAX_FLOW_PIXEL 4
 #define HAL_FLOW_PX4_BOTTOM_FLOW_FEATURE_THRESHOLD 30
 #define HAL_FLOW_PX4_BOTTOM_FLOW_VALUE_THRESHOLD 5000
+#define HAL_PARAM_DEFAULTS_PATH "/etc/arducopter/bebop.parm"
```

Comment:
I agree that we could choose a standard path for this file. I am already using our build system to copy the file at startup with a config to choose between bebop 1 and 2 and that you can see here:
https://github.com/Parrot-Developers/ardupilot/blob/master-alchemy-build/atom.mk#L42
I don't have any objection about the exact path, /etc/apm seems fine. We could accept this PR and then modify this in order to be the same on all linux boards.